src/test/java/org/eclipse/yasson/internal/JsonBindingBuilderTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+
+package org.eclipse.yasson.internal;
+
+import org.glassfish.json.JsonProviderImpl;
+import org.junit.jupiter.api.Test;
+
+import javax.json.bind.Jsonb;
+import javax.json.bind.JsonbConfig;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class JsonBindingBuilderTest {
+
+    @Test
+    public void testMultipleCallsToBuildWithoutChangesReturnTheSameInstance() {
```

Comment:
lets write tests that are more similar to how our users interact with Yasson/JSONB. In this case, you are doing `new JsonBindingBuilder()` which is from a Yasson internal package and users should never be using.
Instead, lets do things like `JsonbBuilder.create()` or `JsonbBuilder.newBuilder().build()`